alembic/versions/2023_01_12_0141-20321580ea8a_add_status_and_mode_attributes_to_node.py

```diff
@@ -0,0 +1,35 @@
+"""Add status and mode attributes to Node
```

Comment:
what's this file

Reply:
Since I'm adding attributes, the metadata tables needed to be updated so I did an alembic revision. It's described here in the contributing doc: https://github.com/DataJunction/dj/blob/main/CONTRIBUTING.rst#database-migrations